src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -177,21 +182,21 @@ const EncounterList: React.FC<EncounterListProps> = ({
                 onClick={e => {
                   e.preventDefault();
                   if (actionItem.mode == 'edit') {
-                    launchFormInEditMode(
+                    launchEncounterForm(
                       applyFormIntent(actionItem.intent, getForm(actionItem.form.package, actionItem.form.name)),
                       actionItem.encounterUuid,
-                      forceComponentUpdate,
                     );
                   } else if (actionItem.mode == 'enter') {
                     launchForm(
```

Comment:
@larslemos don't we need to edit this as well?

Reply:
Thank you for raising this, I was making use of the additional hook, existing at parent level.